folksonomy/api.py

```diff
@@ -469,7 +469,8 @@ async def product_tag_update(response: Response,
             detail=re.sub(r'.*@@ (.*) @@\n.*$', r'\1', e.pgerror)[:-1],
         )
     if cur.rowcount == 1:
-        return "ok"
+        # Return the updated tag instead of just "ok"
+        return product_tag
```

Comment:
This is a change in the API we must ensure no depending app will crash because of this…

One way to avoid this is to add a parameter to ask for the product_tag as return value, and return "ok" otherwise.

It might be that mobile app only use the response http status code, in which case it is ok, but please ask them first (opening an issue in their github repository).